harness/determined/cli/resource_pool.py

```diff
@@ -0,0 +1,166 @@
+
+    print(
+        f'added bindings between the resource pool "{args.pool_name}" '
+        f"and the following workspaces: {workspaces_string}"
+    )
+    return
+
+
+@authentication.required
+def remove_binding(args: Namespace) -> None:
+    body = bindings.v1UnbindRPFromWorkspaceRequest(
+        resourcePoolName=args.pool_name,
+        workspaceNames=args.workspace_names,
+    )
+    bindings.delete_UnbindRPFromWorkspace(
+        setup_session(args), body=body, resourcePoolName=args.pool_name
+    )
+
+    workspaces_string = get_workspaces_string(args.workspace_names)
+    print(
+        f'removed bindings between the resource pool "{args.pool_name}'
+        f"and the following workspaces: {workspaces_string}"
+    )
+    return
+
+
+@authentication.required
+def replace_bindings(args: Namespace) -> None:
+    body = bindings.v1OverwriteRPWorkspaceBindingsRequest(
+        resourcePoolName=args.pool_name,
+        workspaceNames=args.workspace_names,
+    )
+    bindings.put_OverwriteRPWorkspaceBindings(
+        setup_session(args), body=body, resourcePoolName=args.pool_name
+    )
+
+    workspaces_string = get_workspaces_string(args.workspace_names)
+    print(
+        f'replaced bindings of the resource pool "{args.pool_name}'
+        f"with those to the following workspaces: {workspaces_string}"
+    )
+    return
+
+
+@authentication.required
+def list_workspaces(args: Namespace) -> None:
+    session = setup_session(args)
+    resp = bindings.get_ListWorkspacesBoundToRP(session, resourcePoolName=args.pool_name)
+    if resp.workspaceIds is None or len(resp.workspaceIds) == 0:
+        print("resource pool has no assignments")
+        return
+
+    print("ids are:", resp.workspaceIds)  # TODO: convert to names
+    workspace_names = ""
+    workspaces_set = set(resp.workspaceIds)
+    workspaces = bindings.get_GetWorkspaces(session)
+    for workspace in workspaces.workspaces:
+        if workspace.id in workspaces_set:
+            workspace_names += workspace.name + ", "
+
+    if workspace_names != "":
+        workspace_names = workspace_names[:-2]
+
+    headers = ["resource pool", "workspaces"]
+    values = [[args.pool_name, workspace_names]]
+    render.tabulate_or_csv(headers, values, False)
+    return
+
+
+args_description = [
+    Cmd(
+        "resource-pool rp",
+        None,
+        "manage resource pools",
+        [
+            Cmd(
+                "bindings",
+                None,
+                "manage resource pool bindings",
+                [
+                    Cmd(
+                        "add",
+                        add_binding,
+                        "add a resource-pool-to-workspace binding",
+                        [
+                            Arg("pool_name", type=str, help="name of the resource pool to bind"),
+                            Arg(
+                                "workspace_names",
+                                nargs=ONE_OR_MORE,
```

Comment:
I don't actually know how this works (and am apparently too lazy to find out). My reading of this is that both `pool_name` and `workspace_names` are positional arguments. Does this mean that calls will look like:
```shell
det rp bindings add resource_pool workspace1 workspace2 workspace3
```

Reply:
yeah exactly - one or more means that you can do:
`det rp bindings add resource_pool workspace1` 
or
`det rp bindings add resource_pool workspace1 workspace2`

Reply:
That's really nice!